tests/system/alerts/test_alerts.py

```diff
@@ -212,8 +212,9 @@ def test_drift_detection_alert(self):
                 model_endpoint.metadata.uid, result_name, model_endpoint.metadata.name
             )
         )
-
-        time.sleep(5)
+        # wait for the event to be processed by the writer, changed as part of adding tsdb target with flush
+        # configuration of 30 seconds
+        time.sleep(35)
```

Comment:
Elsewhere in the test, this is dealt with using the config value directly, i.e.:
```suggestion
        time.sleep(5 + mlconf.model_endpoint_monitoring.writer_graph.flush_after_seconds)
```